galleries/team/metadata.yml

```diff
@@ -169,3 +169,8 @@ title: Cristian Garcia
 github: cgarciae
 projects: ['elegy', 'treex', 'lux']
 ---
+name: pamphile_roy.jpg
+ title: Pamphile Roy
```

Comment:
1 space indentation is the problem here I think

Reply:
Ooops thanks, it was the copy paste.